.mergify.yml

```diff
@@ -6,7 +6,23 @@
 pull_request_rules:
   - name: automatic merge (squash) on CI success
     conditions:
-      - status-success=Travis CI - Pull Request
+      - check-success=Travis CI - Pull Request
+      - check-success=rustfmt
```

Comment:
Ouch.  There's not a single roll-up status for ALL the github actions?  Listing out each individual action like this will cause automerge to break all the time

Reply:
Unfortunately no -- they even say it's not possible in their docs: https://docs.mergify.io/conditions.html?highlight=check%20success#validating-all-status-check

> There is no such thing as “every status check” in GitHub.

Reply:
on the flip side, we can do something like `check-success~=build` but they recommend against it

Reply:
That's really shitty.  Buildkite solves this by having a roll-up build status that only goes green when the associated jobs go green.  It's disappointing that github actions didn't do that as as well

Reply:
Can we have a single github action that depends on all the other actions?

Reply:
that's a good idea -- let me check and see